test/evergreen/compatibility_test_for_releases.sh

```diff
@@ -52,45 +52,138 @@ build_branch()
             ./configure $config && make -j $(grep -c ^processor /proc/cpuinfo)) > /dev/null
 }
 
+#############################################################
+# get_config_file_name:
+#       arg1: branch name
+#############################################################
+get_config_file_name()
+{
+    local file_name=""
+    branch_name=$1
+    if [ "${wt_standalone}" = true ] || [ $older = true ] ; then
+        file_name="${branch_name}/test/format/CONFIG_default"
+        echo $file_name
+        return
+    fi
+    release=$(echo $branch_name |  cut -c9-)
+    file_name="CONFIG_${release}"
+
+    # Special case to generate develop branch CONFIG
+    if [ -z "$release" ] && [ ${branch_name} == "develop" ] ; then
+        release="develop"
+        file_name="CONFIG_${release}"
+    fi
+    echo $file_name
+}
+
+#############################################################
+# create_configs:
+#       arg1: branch name
+#############################################################
+create_configs()
+{
+    branch_name=$1
+
+    file_name=$(get_config_file_name $branch_name)
+
+    if [ -f $file_name ] ; then
+        return
+    fi
+
+    echo "##################################################" >> $file_name
+    echo "runs.type=row" >> $file_name              # Temporarily disable column store tests
+    echo "btree.prefix=0" >> $file_name             # Prefix testing isn't portable between releases
+    echo "cache=80" >> $file_name                   # Medium cache so there's eviction
+    echo "checksum=on" >> $file_name
+    echo "checkpoints=1"  >> $file_name             # Force periodic writes
+    echo "compression=snappy"  >> $file_name        # We only built with snappy, force the choice
+    echo "data_source=table" >> $file_name
+    echo "huffman_key=0" >> $file_name              # Not supoprted by newer releases
+    echo "in_memory=0" >> $file_name                # Interested in the on-disk format
+    echo "leak_memory=1" >> $file_name              # Faster runs
+    echo "logging=1" >> $file_name                  # Test log compatibility
+    echo "logging_compression=snappy" >> $file_name # We only built with snappy, force the choice
+    echo "rows=1000000" >> $file_name
+    echo "salvage=0" >> $file_name                  # Faster runs
+    echo "timer=4" >> $file_name
+    echo "verify=0" >> $file_name                   # Faster runs
```

Comment:
We should leave verify turned on -- not sure if it was turned off before? -- but this is a compatibility test, so doing a verify is the right thing to do.

Reply:
Yes, it was turned off before. But I turned on now and all the test passes.